content/03.results.md

```diff
@@ -23,14 +23,14 @@ Along with the patient tumors, the Portal contains a small number of human tumor
 
 Each of the available samples contains summarized gene expression data from either single-cell or single-nuclei RNA sequencing.
 However, some samples also include additional data, such as quantified expression data from tagging cells with antibody-derived tags (ADT), such as CITE-seq antibodies [@doi:10.1038/nmeth.4380], or multiplexing samples with hashtag oligonucleotides (HTO) [@doi:10.1186/s13059-018-1603-1] prior to sequencing.
-Out of the 518 samples, 96 have associated CITE-seq data, and 19 have associated multiplexing data.
+Out of the over 700 samples, 95 have associated CITE-seq data, and 35 have associated multiplexing data.
 In some cases, multiple libraries from the same sample were collected for additional sequencing, either for bulk RNA-seq or spatial transcriptomics.
-Specifically, 118 samples on the Portal were sequenced using bulk RNA-seq and 94 samples were sequenced using spatial transcriptomics.
+Specifically, 182 samples on the Portal were sequenced using bulk RNA-seq and 38 samples were sequenced using spatial transcriptomics.
```

Comment:
Where is 182 from? Shouldn't it be 176? https://github.com/AlexsLemonade/scpca-paper-figures/blob/main/figures/pdfs/Fig1B_modality-summary.pdf

Reply:
> Where is 182 from? Shouldn't it be 176? https://github.com/AlexsLemonade/scpca-paper-figures/blob/main/figures/pdfs/Fig1B_modality-summary.pdf

This is mentioned in https://github.com/AlexsLemonade/scpca-paper-figures/pull/277#issue-3009319497. We have 176 single-cell/nuc samples with a bulk library and 6 samples that are bulk but no single-cell/nuc. So 182 total samples with bulk.

Reply:
I think we should add an HTML comment for this as a reminder for later since I don't think this number is recorded in a table anywhere else?

Reply:
Good point! Done in https://github.com/AlexsLemonade/ScPCA-manuscript/pull/169/commits/f2a971715e6e15af2717bc78b9615122018f0a3f